libmscore/layout.cpp

```diff
@@ -3123,17 +3123,20 @@ void Score::layoutLyrics(System* system)
 
 void layoutTies(Chord* ch, System* system, int stick)
       {
+      SysStaff *staff = system->staff(ch->staffIdx());
+      if(!staff->show())
```

Comment:
Some more pedantic style comments:
- `SysStaff* staff` (asterisk position)
- `if (!staff->show())` (whitespace is missing)

Also I see that you have added `fix #282184` to the commit's message but it may be more consistent if you add it to the beginning of the message rather than its description, similar to how it is done in the pull request title.

Otherwise looks good to merge, I believe it is fine to handle the added check inside `layoutTies()`.